nmrfx-processor/src/main/java/org/nmrfx/processor/processing/Processor.java

```diff
@@ -866,14 +911,26 @@ public boolean createNVInMemory(String outputFile, int[] useSizes) {
         return true;
     }
 
+    public boolean createNVInMemory(String outputFile, int[] useSizes, int[] mapToDataset) {
+        createNV(outputFile, useSizes, true);
+        return true;
```

Comment:
why are we passing in mapToDataset, it looks unused, is this calling the wrong function?